_faqs/faq-12.md

```diff
@@ -0,0 +1,8 @@
+---
+question: I want to cite RIOT. Which reference should I use?
+---
+
+Please consider the following references for citation.
+
+  * Emmanuel Baccelli, Cenk G&uuml;ndo&gbreve;an, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdf/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
```

Comment:
`&gbreve;` is not rendering correctly for me
![error_faq](https://user-images.githubusercontent.com/5381296/109477191-072d3700-7a78-11eb-89f2-433878e2aa9f.png)

Reply:
On the other hand, using ğ directly works

Reply:
just pushed a fix.

Reply:
That solves it